go.mod

```diff
@@ -81,6 +81,8 @@ require (
 	sigs.k8s.io/yaml v1.3.0
 )
 
+require github.com/mesosphere/toml-merge v0.2.0
```

Comment:
Can we place this in the k0s require section?

Reply:
oh yes